app/models/foreman_tasks/task/dynflow_task.rb

```diff
@@ -41,12 +41,18 @@ def execution_plan
       @execution_plan ||= ForemanTasks.dynflow.world.persistence.load_execution_plan(external_id)
     end
 
+    def label
```

Comment:
I think it's fine - this one can just remove the initialization code, and the other one includes the 'UI' adaption part.